### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by kcelentano**
@phonxaychampasouk 

- set up dependencies, webpack, and express server
- refactored database schema, seed script, and initial API request from mySQL to mongoDB

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
+}
```
Comment (phonxaychampasouk) [inline .babelrc]: Looks like there needs to be a newline at the end of this file.

File: database/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  multipleStatements: true
+});
+
+connection.connect((err) => {
+  if(!err) {
+    console.log('Connected to database');
+  } else {
+    console.log('Unable to connect to database. ', err);
+  }
+})
```
Comment (phonxaychampasouk) [inline database/index.js]: Another new line, this should be an error that eslint should find.

File: public/bundle.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (phonxaychampasouk) [inline public/bundle.js]: Looks like you may have not put your bundle.js in your GitIgnore.

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<html>
+  <head>
+  </head>
+  <body>
+    <div id="app"></div>
+    <script src="bundle.js"></script>
+  </body>
+</html>
```
Comment (phonxaychampasouk) [inline public/index.html]: New Line.

File: server/index.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const app = express();
+const port = 3003;
+
+app.get('/', (req, res) => res.send('Hello World!'));
+
+app.use(express.static('public'));
+
+app.listen(port, () => console.log(`Example app listening on port ${port}!`));
```
Comment (phonxaychampasouk) [inline server/index.js]: New Line.

File: webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    rules: [
+      { test: /\.(js}jsx)$/, exclude: /node_modules/, loader: 'babel-loader' },
+    ]
+  },
+  resolve: {
+    extensions: ['js', '.jsx'],
+  },
+  devtool: 'inline-source-map',
+  mode: 'development',
+}
```
Comment (phonxaychampasouk) [inline webpack.config.js]: New Line.

Comment (phonxaychampasouk) [inline webpack.config.js]:
```suggestion
}

```

File: .gitignore [hunk 7]
```diff
@@ -24,3 +24,4 @@ coverage
 build/
 dist/
 
+public/bundle.js
```
Comment (phonxaychampasouk) [inline .gitignore]: Disregard if I am wrong but I wonder if your .gitignore should have also been ignored upon push.

File: database/index.js [hunk 8]
```diff
@@ -1,16 +1,47 @@
...
+  monthVisited: String,
+  hereFor: String,
+  votes: Number,
+  device: String,
+  travelerType: String,
+  language: String,
+  rating: Number,
+  serviceRating: Number,
+  foodRating: Number,
+  valueRating: Number,
```
Comment (phonxaychampasouk) [inline database/index.js]: Looks great!

File: database/seed.js [hunk 9]
```diff
@@ -0,0 +1,57 @@
+const faker = require('faker');
+const Review = require('../database/index.js');
+
+const hereForArr = ['', 'Breakfast', 'Brunch', 'Lunch', 'Dinner', 'Dessert', 'Coffee or tea', 'Snacks', 'Drinks', 'Late night food', 'Other'];
+const travelerTypeArr = ['Couples', 'Family', 'Friends', 'Business', 'Solo'];
```
Comment (phonxaychampasouk) [inline database/seed.js]: I see you put an empty string in your hereforArr but not one in your travelerTypeArr.  Is the empty string indicating an option that is not filled in by the reviewer?

File: database/seed.js [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+// Stretch goal: Make it so same user can't leave multiple reviews
+// and highest userId equals total number of users minus 1
+
+const seed = () => {
+  // generate data for 100 restaurants
+  for (let i = 0; i < 100; i += 1) {
+    // generate between 0 and 200 reviews for each restaurant
+    const reviewNum = Math.floor(Math.random() * 200);
+    const claimed = faker.random.boolean();
+    const restaurantName = `${faker.random.word()} Restaurant`;
```
Comment (phonxaychampasouk) [inline database/seed.js]: Love the random.word() + 'Restaurant'.

File: .babelrc (hunk 1 above)
Comment (kcelentano) [reply .babelrc]: Good catch!

File: server/index.js [hunk 11]
```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 const path = require('path');
-const Review = require('../database/index.js');
+// const Review = require('../database/index.js');
```
Comment (phonxaychampasouk) [inline server/index.js]: This comment might be able to be deleted if you dont think you weill ever use it again.

File: database/seed.js (hunk 9 above)
Comment (kcelentano) [reply database/seed.js]: Great question! I put in '' to account for the fact that it's an optional field. However, travelerType is also optional -- I need to add '' there too. Thanks for catching!

File: database/seed.js (hunk 10 above)
Comment (kcelentano) [reply database/seed.js]: Thanks! I figured it was the simplest way to randomly generate a name that was believably a restaurant.

File: public/bundle.js (hunk 3 above)
Comment (kcelentano) [reply public/bundle.js]: Hmmm, strange. I'll see what's up here... It's listed as "public/bundle.js" in .gitignore, but this obviously didn't work. Do you have any thoughts as to why? No worries if not -- I can research.

File: public/index.html (hunk 4 above)
Comment (kcelentano) [reply public/index.html]: Thanks!

File: .gitignore (hunk 7 above)
Comment (kcelentano) [reply .gitignore]: Fixed!

File: public/bundle.js (hunk 3 above)
Comment (kcelentano) [reply public/bundle.js]: I figured it out -- the file had been tracked prior to my adding it to .gitignore, so I had to manually remove it via git rm --cached public/bundle.js

**Pull request merged**